rest/config.go

```diff
@@ -812,6 +829,27 @@ func (self *ServerConfig) MergeWith(other *ServerConfig) error {
 	return nil
 }
 
+func (sc *ServerConfig) Redacted() (*ServerConfig, error) {
+	var config ServerConfig
+
+	err := base.DeepCopyInefficient(&config, sc)
+	if err != nil {
+		return nil, err
+	}
+
+	for i := range config.Databases {
+		config.Databases[i], err = config.Databases[i].Redacted()
+		if err != nil {
+			return nil, err
+		}
+		for j, _ := range config.Databases[i].Replications {
+			config.Databases[i].Replications[j] = config.Databases[i].Replications[j].Redacted()
+		}
```

Comment:
We should move this into `DbConfig.Redacted()` so it's applied on `/db/_config` too.